matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -281,7 +281,7 @@ internal class LocalEchoEventFactory @Inject constructor(@UserId private val use
     }
 
     private fun dummyEventId(): String {
-        return "$LOCAL_ID_PREFIX${UUID.randomUUID()}"
+        return "${LocalEcho.PREFIX}${UUID.randomUUID()}"
```

Comment:
Move this method to LocalEcho and rename to `createEventId`

Reply:
Don't want createEventId to be public!

Reply:
it can be internal?